src/components/todoitem.js

```diff
@@ -0,0 +1,6 @@
+export default class TodoItem {
+  constructor(description, completed = false) {
```

Comment:
- [ ] Let's make sure that when a to-do list is marked as completed all the changes are persisted on the LocalStorage as well.

![no-completed](https://user-images.githubusercontent.com/79114233/149261604-e4440dda-3452-4dc6-a67f-3a5c2bbb9b30.gif)